packages/lodestar/src/constants/network.ts

```diff
@@ -159,3 +162,9 @@ export const goodbyeReasonCodeDescriptions: Record<string, string> = {
   250: "Peer score too low",
   251: "Peer banned this node",
 };
+
+/** Until js-libp2p types its events */
```

Comment:
Aren't we maintaining types for that still in libp2p-ts?

Reply:
libp2p-ts types these events but as a string uninon, not an enum. 
```ts
export type ConnectionEvents = 'peer:connect' | 'peer:disconnect';
```
https://github.com/ChainSafe/libp2p-ts/blob/86c1d9eca212194c8a5ce1ea6572a3bd694103e5/types/libp2p/index.d.ts#L95

Reply:
Why not change that there instead of here? 😄

Reply:
Just opened an issue https://github.com/ChainSafe/libp2p-ts/issues/21